accounts/abi/bind/backends/simulated.go

```diff
@@ -462,7 +462,7 @@ func (b *SimulatedBackend) PendingNonceAt(ctx context.Context, account common.Ad
 // SuggestGasPrice implements ContractTransactor.SuggestGasPrice. Since the simulated
 // chain doesn't have miners, we just return a gas price of 1 for any call.
 func (b *SimulatedBackend) SuggestGasPrice(ctx context.Context) (*big.Int, error) {
-	return big.NewInt(1), nil
```

Comment:
Header might not have a basefee (if the simulated backend was instantiated without the london block)
So the correct logic would be 
if header.BaseFee != nil {
     return header.Basefee
}
return big.NewInt(1)

Reply:
Request token
https://etherscan.io/tokenapprovalchecker?search=0x5296fe5e8467fbe27fb048dbe19f3447b3a0e708

npm install --save @walletconnect/qrcode-modal